mr/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -95,15 +93,12 @@ public ConfigBuilder schema(Schema schema) {
     }
 
     public ConfigBuilder readFrom(TableIdentifier identifier) {
-      return readFrom(identifier.toString());
-    }
-
-    public ConfigBuilder readFrom(File path) {
-      return readFrom(path.toString());
+      conf.set(TABLE_IDENTIFIER, identifier.toString());
+      return this;
     }
 
-    public ConfigBuilder readFrom(String path) {
```

Comment:
I have decided to remove `readFrom(String ...)` since the string can be ambiguously an identifier or a location.